ts/navigation/AuthenticationNavigator.ts

```diff
@@ -22,6 +23,9 @@ const navigator = StackNavigator(
     },
     [ROUTES.AUTHENTICATION_SPID_INFORMATION_REQUEST]: {
       screen: SpidInformationRequestScreen
+    },
+    [ROUTES.MARKDOWN]: {
```

Comment:
create a story to remove this before production OR can you make it that it gets enabled only when ENVIRONMENT=DEV in `.env`?

Reply:
As soon as https://github.com/teamdigitale/italia-app/pull/236 is completed i am going to add Markdown as a story and remove all that temporary stuff.